tools/docker-compose/ansible/roles/sources/templates/docker-compose.yml.j2

```diff
@@ -40,6 +41,8 @@ services:
       - "../../docker-compose/_sources/database.py:/etc/tower/conf.d/database.py"
       - "../../docker-compose/_sources/websocket_secret.py:/etc/tower/conf.d/websocket_secret.py"
       - "../../docker-compose/_sources/local_settings.py:/etc/tower/conf.d/local_settings.py"
```

Comment:
well this is confusing that we had 2 paths (on the host machine) where local_settings.py could be - either in `awx/settings/local_settings.py` or in `tools/docker-compose/_sources/local_settings.py`